pkg/build/controller/image_change_controller.go

```diff
@@ -61,6 +61,7 @@ func (c *ImageChangeController) HandleImageRepo(repo *imageapi.ImageStream) erro
 			from           *kapi.ObjectReference
 			shouldBuild    = false
 			triggeredImage = ""
+			next           = ""
```

Comment:
instead of doing this i think you can use the triggeredImage variable that already exists in the scope you need.